tests/logging/logging_resources_test.py

```diff
@@ -120,6 +120,7 @@ def test_check_file_handlers(tmp_path: Path):
     hdlr = BeamlimeFileHandler(tmp_file)
     logger.addHandler(hdlr)
     assert hdlr in logger.handlers
+    hdlr.close()
```

Comment:
Only `Windows` complained when I tried to remove the file while it's being used by the handler.